### PR TITLE
PEP8 fixes, copyright update, put version in setup.py changed from irow to iat

### PR DESCRIPTION
**Pull request by iskandr**

### CODE REVIEW

File: gtfparse/attribute_parsing.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2015. Mount Sinai School of Medicine
+# Copyright (c) 2015-2016. Mount Sinai School of Medicine
```
Comment (tavinathanson) [inline gtfparse/attribute_parsing.py]: 2015-2016? First I've seen that!

File: gtfparse/create_missing_features.py [hunk 2]
```diff
@@ -98,11 +96,11 @@ def create_missing_features(
 
             # assume that seqname and strand are the same for all other
             # entries in the GTF which shared this unique ID
-            feature_values["seqname"][i] = group["seqname"].irow(0)
-            feature_values["strand"][i] = group["strand"].irow(0)
+            feature_values["seqname"][i] = group["seqname"].iat[0]
```
Comment (tavinathanson) [inline gtfparse/create_missing_features.py]: So this works with `iat` because it's a `Series`?

Comment (tavinathanson) [inline gtfparse/create_missing_features.py]: Why the change?

**Comment by tavinathanson** [comment]
LGTM

File: gtfparse/attribute_parsing.py (hunk 1 above)
Comment (iskandr) [inline gtfparse/attribute_parsing.py]: Not sure if we need it, but nominally seems to extend the copyright forward to last edited date: http://programmers.stackexchange.com/questions/124687/copyright-statements-in-source-code

File: gtfparse/create_missing_features.py (hunk 2 above)
Comment (iskandr) [inline gtfparse/create_missing_features.py]: `irow` is deprecated, throws up a warning, and seems to be slower.
